### PR TITLE
LPS-193361

### PR DESCRIPTION
**Pull request by G-Sa**

### CODE REVIEW

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 1]
```diff
@@ -578,8 +577,8 @@ public ObjectDefinition putObjectDefinition(
 					serviceBuilderObjectField.isMetadata() ||
 					!serviceBuilderObjectField.isSystem());
 			serviceBuilderObjectRelationships.removeIf(
-				serviceBuilderObjectRelationship ->
-					!serviceBuilderObjectRelationship.isSystem());
+				serviceBuilderObjectRelationship -> !GetterUtil.getBoolean(
```
Comment (gabrielwas) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:580]: No need for the GetterUtil

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 2]
```diff
@@ -589,7 +588,8 @@ public ObjectDefinition putObjectDefinition(
 					objectRelationship.getSystem()));
 			serviceBuilderObjectFields.removeIf(ObjectFieldModel::isSystem);
 			serviceBuilderObjectRelationships.removeIf(
-				ObjectRelationshipModel::isSystem);
+				serviceBuilderObjectRelationship -> GetterUtil.getBoolean(
```
Comment (gabrielwas) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:591]: same here

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 3]
```diff
@@ -291,15 +291,15 @@ public ObjectRelationship deleteObjectRelationship(
 				"Edge object relationships cannot be deleted");
 		}
 
+		_validateInvokerBundle(
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:294]: go back to the former way

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 4]
```diff
@@ -649,16 +649,20 @@ public ObjectDefinition putObjectDefinition(
 			_objectLayoutLocalService.deleteObjectLayouts(objectDefinitionId);
 		}
 
+		ObjectRelationship[] selectedObjectRelationships =
```
Comment (gabrielwas) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:652]: no need a new variable

**Review by gabrielwas — CHANGES_REQUESTED** [review]
Pls, see the comments

**Comment by G-Sa** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
resent in https://github.com/liferay-objects/liferay-portal/pull/2601

**Pull request closed without merging**